psh/ps/ps.c

```diff
@@ -136,29 +136,42 @@ static int psh_ps(int argc, char **argv)
 				tcnt -= j - i - 1;
 			}
 		}
-		printf("%8s %8s %2s %5s %5s %6s %7s %6s %3s %-21s\n", "PID", "PPID", "PR", "STATE", "%CPU", "WAIT", "TIME", "VMEM", "THR", "CMD");
+		printf("%8s %8s %2s %5s %5s %6s %11s %6s %3s %-17s\n", "PID", "PPID", "PR", "STATE", "%CPU", "WAIT", "TIME", "VMEM", "THR", "CMD");
 	}
 	else {
-		printf("%8s %8s %2s %5s %5s %6s %7s %6s %-25s\n", "PID", "PPID", "PR", "STATE", "%CPU", "WAIT", "TIME", "VMEM", "CMD");
+		printf("%8s %8s %2s %5s %5s %6s %11s %6s %-21s\n", "PID", "PPID", "PR", "STATE", "%CPU", "WAIT", "TIME", "VMEM", "CMD");
 	}
 
 	qsort(info, tcnt, sizeof(threadinfo_t), cmp);
 
 	for (i = 0; i < tcnt; i++) {
-		info[i].cpuTime /= 10000;
-		h = info[i].cpuTime / 3600;
-		m = (info[i].cpuTime - h * 3600) / 60;
 		psh_prefix(10, info[i].wait, -6, 1, buff);
-		printf("%8u %8u %2d %5s %3u.%u %5ss %4u:%02u ", info[i].pid, info[i].ppid, info[i].priority, (info[i].state) ? "sleep" : "ready",
-			info[i].load / 10, info[i].load % 10, buff, h, m);
-
+		printf("%8u %8u %2d %5s %3u.%u %5ss ", info[i].pid, info[i].ppid, info[i].priority, (info[i].state) ? "sleep" : "ready",
+			info[i].load / 10, info[i].load % 10, buff);
+
+		s = info[i].cpuTime / 1000000;
```

Comment:
We want to round the result down? If not:
```suggestion
		s = (info[i].cpuTime + 555555) / 1000000;
```

Reply:
I've added rounding to the nearest whole number.